mmv1/products/backupdr/BackupPlan.yaml

```diff
@@ -236,3 +243,7 @@ properties:
                   description: |
                     The hour of the day (1-24) when the window ends, for example, if the value of end hour of the day is 10, that means the backup window end time is 10:00.
                     The end hour of the day should be greater than the start
+  - name: 'logRetentionDays'
```

Comment:
Please add `immutable: true` given the field is not updatable (based on the [comment](https://github.com/GoogleCloudPlatform/magic-modules/pull/14622/files#r2241542950))